ports/mimxrt/Makefile

```diff
@@ -54,12 +55,14 @@ INC += -I$(TOP)/$(MCU_DIR)/project_template
 INC += -I$(TOP)/lib/tinyusb/src
 INC += -I$(TOP)/lib/tinyusb/hw
 INC += -I$(TOP)/lib/tinyusb/hw/bsp/teensy_40
+INC += -I$(TOP)/lib/oofatfs
```

Comment:
please add in a sorted order, ie just before `lib/tinyusb/src` (yes, even if the others are not strictly ordered :)

Reply:
done. I have ordered all of the "lists" alphabetically now.